homeassistant/components/here_travel_time/const.py

```diff
@@ -0,0 +1,113 @@
+"""Constants for the HERE Travel Time integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+from typing import TypedDict
+
+from homeassistant.const import (
+    CONF_UNIT_SYSTEM,
+    CONF_UNIT_SYSTEM_IMPERIAL,
+    CONF_UNIT_SYSTEM_METRIC,
+)
+
+
+class HERERoutingData(TypedDict):
+    """Routing information calculated from a herepy.RoutingResponse."""
+
+    ATTR_ATTRIBUTION: str | None
+    ATTR_DURATION: float
+    ATTR_DURATION_IN_TRAFFIC: float
+    ATTR_DISTANCE: float
+    ATTR_ROUTE: str
+    ATTR_ORIGIN: str
+    ATTR_DESTINATION: str
+    ATTR_ORIGIN_NAME: str
+    ATTR_DESTINATION_NAME: str
+
+
+@dataclass
```

Comment:
These data classes don't exactly seem like constants, though I get the point they are simple types you want to share

Reply:
Do you have a more fitting place in mind? Maybe a `utils.py` or `dataclasses.py`?

Reply:
I did a quick search and `model.py` seems like a common choice.